charts/observation-service/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
```

Comment:
there is a helmignore file in the root of this project, you can add a symlink to that one.

Reply:
I still see the new file., Is this replaced with the symlink?  

you can remove this file and add a symlink via `ln -s ../../.helmignore .`